addons/mail/static/src/messaging/component/notification_popover/notification_popover.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <t t-name="mail.messaging.component.NotificationPopover" owl="1">
+        <div class="o_NotificationPopover">
+            <t t-foreach="notifications" t-as="notification" t-key="notification.localId">
+                <div class="o_NotificationPopover_notification">
+                    <i class="o_NotificationPopover_icon" t-att-class="iconClass" t-att-title="iconTitle" role="img"/>
+                    <span class="o_NotificationPopover_partnerName" t-esc="notification.partner.nameOrDisplayName"/>
+                </div>
```

Comment:
The 2 inner classes should contain _notification somewhere in their name?

Reply:
Don't see the gain of prefixing part names with `_notification`, since this component is about **Notification**Popover